src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.cs

```diff
@@ -303,7 +305,20 @@ internal async Task<bool> ApplyChangedSettingsAsync(CancellationToken cancellati
             // Unknown exception.
             catch (Exception ex)
             {
-                MessageHelper.ShowErrorMessage(Resources.ShowError_ApplySettingFailed, Resources.ErrorDialogBoxTitle);
+                if (ex is RemoteInvocationException remoteException
+                && remoteException.DeserializedErrorData is CommonErrorData commonError)
```

Comment:
>&&  [](http://example.com/codeflow?start=16&length=3)

NIT: The coding convention we use is to have the logical operator at the end of each line. See line 270 for example.